### ISSUE TITLE
Bellman Ford[C++]

### ISSUE THREAD

**Issue by Chirag-Mathur** [iiitv/algos#723]
#### This issue is about [RedBlackTree](https://github.com/iiitv/algos).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by Chirag-Mathur** [comment]
@iiitv/project-maintainers I want to work on this issue

**Comment by yashshah2820** [comment]
Is it about bellman ford or red black tree?

**Comment by Chirag-Mathur** [comment]
It is about bellman ford algorithm of graph. It's C++ implementation.

**Comment by Chirag-Mathur** [comment]
@iiitv/project-maintainers I am willing to work on this issue.

**Comment by yashshah2820** [comment]
Don't mention the commits of other PR with this issue create a separate PR for this issue

**Comment by Vijay-K-2003** [comment]
- [ ] Assign this issue to me (I want to work on this).
- [ ] I ensure that I am not already assigned to 2 or more issues.
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

Kindly Assign me this issue under Contribute-a-thon.

**Comment by Vijay-K-2003** [comment]
- [ ] Assign this issue to me (I want to work on this).
- [ ] I ensure that I am not already assigned to 2 or more issues.
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

Please Assign me this issue under Contribute-a-thon.

**Comment by Vijay-K-2003** [comment]
### Example
![image](https://user-images.githubusercontent.com/75903092/135646157-3d4da658-afb5-44bd-a410-d051fb2d918a.png)
